share/instruments/instruments.xml

```diff
@@ -13776,10 +13776,18 @@
                   <shortName>Pds. v.</shortName>
                   <description>Highest-pitched member of the viol family.</description>
                   <musicXMLid>strings.viol</musicXMLid>
+                  <StringData>
+                        <frets>13</frets>
+                        <string>55</string>
+                        <string>62</string>
+                        <string>69</string>
+                        <string>74</string>
+                        <string>79</string>
+                  </StringData>
```

Comment:
These pitches are G3, D4, A4, D5, G5. I took these from https://en.wikipedia.org/wiki/Pardessus_de_viole for the 5-string variant of the Pardessus de viole, which is apparently more common than the two 6-string variants.